--- /usr/lib/python2.4/site-packages/mock/backend.py	2010-11-21 12:09:24.000000000 -0500
+++ /usr/lib/python2.4/site-packages/mock/backend.py	2010-11-21 12:08:28.000000000 -0500
@@ -283,6 +283,10 @@
         finally:
             self._umountall()
 
+        # VPX patch
+        # Skip the rest of this setup -- we don't want it.
+        return
+
         # create user
         self._makeBuildUser()
 
